### ISSUE TITLE
Overlap between meta-training and meta-testing?

### ISSUE THREAD

**Issue by atseng17** [dragen1860/MAML-Pytorch#56]
I've noticed that based on the custom dataset the author wrote, there could be overlaps  meta-training and meta-testing, as the classes are chosen in line 106 in `MiniImagenet.py`, so when creating the meta-testset, the classes selected from the meta-trainset could reappear again in the test set. Not sure if im correct.

**Comment by atseng17** [comment]
Oh just found out that the labels in test.cvs do not apprear in train.csv, sorry about the confusion!
